src/selectable-group.js

```diff
@@ -13,7 +13,10 @@ class SelectableGroup extends React.Component {
 		this.state = {
 			isBoxSelecting: false,
 			boxWidth: 0,
-			boxHeight: 0			
+			boxHeight: 0,
+			mouseDownStarted: false,
+			mouseMoveStarted: false,
+			mouseUpStarted: false
```

Comment:
Why do we need to be tracking these mouse events? In what scenario does `mouseUp` fire before `mouseDown`?